.github/workflows/build.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Canary Release
         run: |
           echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' >> .npmrc
-          date=`date +%Y%m%d%H%M%S`
+          yarn changeset version --snapshot canary
```

Comment:
I would use prerelease:
`yarn changeset pre enter ${date}`

See complete doc [here](https://github.com/changesets/changesets/blob/main/docs/prereleases.md)

Reply:
I tried this and it looks promising, however when I do anythong with changeset it wants to bump everything by a major version `changeset status --verbose` tells me everything major, although only three packages are updated with minor. I do not understand this at all here is some explainaition: https://github.com/changesets/changesets/issues/524

Adding the experimental options avoids the major version bump but in the prerelease mode it still wants to add amajor version bump. Really puzzled...

Reply:
I think, I got it working using the following way:
- Remove all the existing changeset
- Copy a dummy changeset doing a patch version increase on all packages
- Enter the prerelease mode with tag the date as @jjtang1985 suggested
- Update the version 
- The changes should be not checked in of course but this is not the case in the pipeline I think